unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/AvatarEditorHUD/Scripts/AvatarEditorHUDController.cs

```diff
@@ -201,45 +254,7 @@ private void LoadOwnedWereables(UserProfile userProfile)
         if (AreWearablesAlreadyLoaded() && IsWearableUpdateInCooldown())
             return;
 
-        lastTimeOwnedWearablesChecked = Time.realtimeSinceStartup;
-
-        loadingWearables = true;
-        CatalogController.RequestOwnedWearables(userProfile.userId)
-                         .Then((ownedWearables) =>
-                         {
-                             ownedWearablesAlreadyLoaded = true;
-                             //Prior profile V1 emotes must be retrieved along the wearables, onwards they will be requested separatedly
-                             this.userProfile.SetInventory(ownedWearables.Select(x => x.id).Concat(thirdPartyWearablesLoaded).ToArray());
-                             LoadUserProfile(userProfile, true);
-                             if (userProfile != null && userProfile.avatar != null)
-                             {
-                                 emotesLoadedAsWearables = ownedWearables.Where(x => x.IsEmote()).ToArray();
-                             }
-                             loadingWearables = false;
-                         })
-                         .Catch((error) =>
-                         {
-                             ownedWearablesRemainingRequests--;
-                             if (ownedWearablesRemainingRequests > 0)
-                             {
-                                 Debug.LogWarning("Retrying owned wereables loading...");
-                                 LoadOwnedWereables(userProfile);
-                             }
-                             else
-                             {
-                                 LoadUserProfile(userProfile, true);
-                                 NotificationsController.i.ShowNotification(new Model
-                                 {
-                                     message = LOADING_OWNED_WEARABLES_ERROR_MESSAGE,
-                                     type = Type.GENERIC,
-                                     timer = 10f,
-                                     destroyOnFinish = true
-                                 });
-
-                                 Debug.LogError(error);
-                                 loadingWearables = false;
-                             }
-                         });
+        RequestOwnedWearablesAsync(CancellationToken.None).Forget();
```

Comment:
You specified this in the original description of the refactor's ticket:

```
- [ ] Change Promises Based consumers to async UniTask but don't provide any cancellation at the moment (it will not cause regression)
```

For this reason I didn't add it. Anyway I think it is very straigh forward to add it and it worth it to do it.

Done in: https://github.com/decentraland/unity-renderer/commit/a68e4cc8ae1582b3e583f4a836629f40efc47099